.circleci/config.yml

```diff
@@ -79,36 +98,69 @@ jobs:
             cd ansible && ansible-playbook -e 'ansible_python_interpreter="/usr/bin/env python"' \
               --check -i localhost, environments.yml
 
-  deploy:
+  setup_uat:
     <<: *container_config
+    environment:
+      - DEPLOY_ENV: uat
```

Comment:
I see some traces of static_uat https://github.com/aeternity/infrastructure/blob/56cfd6326b59f8e91afe53397b7c1d2f5ae11eae/ansible/dump_inventory.py#L16
Does this interact with that in any way?

Reply:
This is leftover, not really related but I removed it in 8affbf0